server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSSendOptionsCheckUtility.java

```diff
@@ -0,0 +1,250 @@
+          put(CICSLexer.MAPPED, ErrorSeverity.WARNING);
+          put(CICSLexer.NOEDIT, ErrorSeverity.WARNING);
+        }
+      };
+
+
+  public CICSSendOptionsCheckUtility(DialectProcessingContext context, List<SyntaxError> errors) {
+    super(context, errors, DUPLICATE_CHECK_OPTIONS);
+  }
+
+  /**
+   * Entrypoint to check CICS Send rule options
+   *
+   * @param ctx ParserRuleContext subclass containing options
+   * @param <E> A subclass of ParserRuleContext
+   */
+  public <E extends ParserRuleContext> void checkOptions(E ctx) {
+    switch (ctx.getRuleIndex()) {
+      case CICSParser.RULE_cics_send_group1:
+        checkGroup1((CICSParser.Cics_send_group1Context) ctx);
+        break;
+      case CICSParser.RULE_cics_send_control_map:
+        checkControl((CICSParser.Cics_send_control_mapContext) ctx);
+        break;
+      case CICSParser.RULE_cics_send_mappingdev:
+        checkMappingdev((CICSParser.Cics_send_mappingdevContext) ctx);
+        break;
+      case CICSParser.RULE_cics_send_page:
+        checkPage((CICSParser.Cics_send_pageContext) ctx);
+        break;
+      case CICSParser.RULE_cics_send_text:
+        checkText((CICSParser.Cics_send_textContext) ctx);
+        break;
+      case CICSParser.RULE_cics_send_text_mapped:
+        checkTextMapped((CICSParser.Cics_send_text_mappedContext) ctx);
+        break;
+      case CICSParser.RULE_cics_send_text_noedit:
+        checkTextNoedit((CICSParser.Cics_send_text_noeditContext) ctx);
+        break;
+      default:
+        break;
+    }
+    checkDuplicates(ctx);
+  }
+
+  @SuppressWarnings("unchecked")
+  private void checkGroup1(CICSParser.Cics_send_group1Context ctx) {
+    checkHasMutuallyExclusiveOptions("LENGTH or FLENGTH", ctx.LENGTH(), ctx.FLENGTH());
+    checkHasMutuallyExclusiveOptions("INVITE or LAST", ctx.INVITE(), ctx.LAST());
+    checkHasMutuallyExclusiveOptions("STRFIELD or ERASE", ctx.STRFIELD(), ctx.ERASE());
+    checkHasMutuallyExclusiveOptions("DEFAULT or ALTERNATE", ctx.DEFAULT(), ctx.ALTERNATE());
+    if (!ctx.DEFAULT().isEmpty() || !ctx.ALTERNATE().isEmpty()) {
+      checkHasMandatoryOptions(ctx.ERASE(), ctx, "ERASE");
+    }
+    checkHasMutuallyExclusiveOptions("STRFIELD or CTLCHAR", ctx.STRFIELD(), ctx.CTLCHAR());
+    checkHasMutuallyExclusiveOptions("CONFIRM or WAIT", ctx.CONFIRM(), ctx.WAIT());
+    checkHasMutuallyExclusiveOptions("CNOTCOMPL or DEFRESP", ctx.CNOTCOMPL(), ctx.DEFRESP());
+    checkHasMutuallyExclusiveOptions("LDC or FMH", ctx.LDC(), ctx.FMH());
+    checkHasMutuallyExclusiveOptions("PASSBK or CBUFF", ctx.PASSBK(), ctx.CBUFF());
+  }
+
+  @SuppressWarnings("unchecked")
+  private void checkControl(CICSParser.Cics_send_control_mapContext ctx) {
+    checkHasExactlyOneOption("CONTROL or MAP", ctx, ctx.CONTROL(), ctx.MAP());
+    checkHasMutuallyExclusiveOptions("ERASE or ERASEAUP", ctx.ERASE(), ctx.ERASEAUP());
+    if (!ctx.DEFAULT().isEmpty() || !ctx.ALTERNATE().isEmpty()) {
```

Comment:
Although it is stated as this is docs, ERASE is not mandatory to use DEFAULT or ALTERNATE 
![image](https://github.com/user-attachments/assets/8ce309da-7d38-4c95-98be-ca37560cf272)